content/en/user-manual/graphics/lighting/shadows.md

```diff
@@ -18,9 +18,23 @@ By default, shadow casting is disabled in PlayCanvas. You have to explicitly ena
 
 ![Model Component][6]
 
-Now you need to specify which graphical objects in your scene cast and receive shadows. By default, all model components receive shadows but do not cast them. Therefore, in order to see an entity cast shadows, select it in the Hierarchy, locate the model component in the Inspector and check the 'Cast Shadows' option.
+Now you need to specify which graphical objects in your scene cast and receive shadows. By default, all render and model components cast and receive shadows. To modify these properties, select the entity in the Hierarchy, locate the render or model component in the Inspector and uncheck the 'Cast Shadows' or 'Receive Shadows' option as required.
```

Comment:
Optional: Would be great to have a screenshot of these options

Reply:
There is a screenshot for this showing the model component. I just slightly improved the text to have up to date defaults.